.eslint-dictionary.json

```diff
@@ -1,201 +1,394 @@
 [
+  "accessor",
+  "adaptor",
+  "adaptors",
+  "addgraphqldatasource",
+  "admingroup",
   "adminui",
+  "aggs",
+  "allowlist",
+  "alphanumerics",
   "amazonaws",
+  "amazoncognito",
+  "amd64",
+  "amplifyadmin",
+  "amplifyapp",
+  "amplifyconfig",
+  "amplifyconfiguration",
+  "amplifyconsole",
+  "amplifyhosting",
```

Comment:
these all look like incorrect camelCasing, shouldn't they be errors?

Reply:
All but `amplifyconsole` are referenced in urls or string literals in the codebase.

Reply:
To clarify, amplify console can be removed here.